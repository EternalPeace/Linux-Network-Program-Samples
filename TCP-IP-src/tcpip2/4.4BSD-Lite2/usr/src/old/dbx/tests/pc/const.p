program consttest (input, output);
const
    PI = 3.14159;
    N = 10;
    MESSAGE = 'this is a test';
    CH = 'x';
type
    Color = (RED, BLUE, GREEN);

procedure p ;
const
    R = RED;
type
    T = integer;
var i : integer;
begin
end;

begin
end.
