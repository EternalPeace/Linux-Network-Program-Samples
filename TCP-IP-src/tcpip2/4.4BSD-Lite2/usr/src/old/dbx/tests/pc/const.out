reading symbolic information ...
const PI = 3.14159;
const N = 10;
const MESSAGE = 'this is a test';
const CH = 'x';
const R = RED;
p.R = RED 
CH = 'x' 
MESSAGE = 'this is a test' 
N = 10 
PI = 3.14159 

